command/ability_check.go

```diff
@@ -1,4 +1,4 @@
-package main
+package cmd
```

Comment:
Could this file and the rest remain as `package main` and instead be moved into `cmd/pd/`?